packages/integrations/netlify/README.md

```diff
@@ -129,14 +126,31 @@ import netlify from '@astrojs/netlify/functions';
 export default defineConfig({
   output: 'server',
   adapter: netlify({
-    builders: true
+    builders: true,
   }),
 });
 ```
 
-On-demand Builders are only available with the `@astrojs/netlify/functions` adapter and are not compatible with Edge Functions.
+On-demand Builders optionally let you return a [TTL](https://docs.netlify.com/configure-builds/on-demand-builders/#time-to-live-ttl) (time-to-live) value for the cached page. The astro adapter allows you to set a default TTL value for all pages, or a custom TTL value for specific pages via a function call.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify/functions';
 
+export default defineConfig({
+  output: 'server',
+  adapter: netlify({
+    builders: {
+      ttl(event) {
+        return event.path.startsWith('/blog') ? 3600 : undefined;
+      },
+    },
+  }),
+});
+```
 
+On-demand Builders are only available with the `@astrojs/netlify/functions` adapter and are not compatible with Edge Functions.
```

Comment:
The sentence above also starts with "On-demand Builders" and I think we should call a bit more attention to it, so I changed it a bit adding "Note that" so people notice this part more!
```suggestion
Note that On-demand Builders are only available with the `@astrojs/netlify/functions` adapter and are not compatible with Edge Functions.
```

Reply:
@Yan-Thomas Great suggestions, thanks 🙏